topics/vanilla-js/index.md

```diff
@@ -0,0 +1,11 @@
+---
+aliases: vanilla-javascript, vanillajs, js-vanilla
+created_by: Brendan Eich
+display_name: Vanilla JavaScript
+related: javascript, ecmascript
+released: December 04 1995
+short_description: Vanilla JavaScript is the native JavaScript language included in the browser.
+topic: vanilla-js
+wikipedia_url: https://en.wikipedia.org/w/index.php?title=Vanilla_JavaScript&redirect=no
```

Comment:
This url doesn't seem useful since it's just a wiki redirect page. Thoughts on removing it?
```suggestion
```

Reply:
Good idea. Should we replace it with the page it's redirected to, or just not have it at all?

Reply:
It just redirects to JavaScript, so I’m thinking that we should just remove for now. Feel free to accept my suggestion.

Reply:
I BELIEVE THAT SIMPLY REMOVING IT IS REASONABLE